analysis_for_grading.R

```diff
@@ -0,0 +1,157 @@
+#github was very uncooperative, we all did pretty equal work though 
+#1)our figures provide good evidence based on the cases per day the the disease originated in country X. 
```

Comment:
This is not specific enough. You need to point to the evidence from the graph and say why the cases matter.
-0.5

Reply:
Also it's unclear which graph you are referring to